project/build.scala

```diff
@@ -1,11 +1,12 @@
 import sbt._
 
+
 object Build extends sbt.Build {
-  lazy val geospace= Project( "geospace-proj", file(".") ).
-    settings(BuildSettings.buildSettings : _*).
+  lazy val geospace = Project( "geospace-root", file("."), settings = BuildSettings.rootSettings).
+    settings(BuildSettings.buildSettings: _*).
     aggregate(geoLibrary, geospaceHttp)
 
-  private def p(name: String, settings : { def settings: Seq[Setting[_]] }, dependencies: ClasspathDep[ProjectReference]*) =
+  def p(name: String, settings : { def settings: Seq[Setting[_]] }, dependencies: ClasspathDep[ProjectReference]*) =
```

Comment:
Why make it public? Is something outside the Build object actually using this method?